src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  margin-top: 0;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-input {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.big-input::placeholder {
+  font-weight: 300;
+  font-family: inherit;
+  font-size: inherit;
+  color: #3d4e61;
+}
+
+.big-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.big-image {
+  position: absolute;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small-input {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.small-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.small-image {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  height: 11px;
+}
+
+.small-input::placeholder {
+  font-weight: 400;
+  font-family: inherit;
+  font-size: inherit;
+  color: #3d4e61;
+}
```

Comment:
check the 2nd topic in the checklist

Reply:
I don't know if I fixed correctly, please check. Thanks.